sdk/build.gradle

```diff
@@ -25,12 +25,12 @@ ext {
 }
 
 android {
-    compileSdkVersion 22
-    buildToolsVersion '22.0.1'
+    compileSdkVersion 23
+    buildToolsVersion '23.0.1'
```

Comment:
Are you sure we can safely update to 23.0.1 ? I have seen some bugs out in the wild due to them.